x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/DownsampleStep.java

```diff
@@ -89,48 +84,26 @@ public void performAction(
             if (IndexMetadata.DownsampleTaskStatus.SUCCESS.equals(downsampleIndexStatus)) {
                 // Downsample index has already been created with the generated name and its status is "success".
                 // So we skip index downsample creation.
-                logger.warn(
+                logger.info(
                     "skipping [{}] step for index [{}] as part of policy [{}] as the downsample index [{}] already exists",
                     DownsampleStep.NAME,
                     indexName,
                     policyName,
                     downsampleIndexName
                 );
                 listener.onResponse(null);
-            } else {
-                // Downsample index has already been created with the generated name but its status is not "success".
-                // So we fail this step so that we go back to cleaning up the index and try again with a new downsample
-                // index name.
-                downsampleFailed = true;
-                listener.onFailure(
-                    new IllegalStateException(
-                        "failing ["
-                            + DownsampleStep.NAME
-                            + "] step for index ["
-                            + indexName
-                            + "] as part of policy ["
-                            + policyName
-                            + "] because the downsample index ["
-                            + downsampleIndexName
-                            + "] already exists with downsample status ["
-                            + downsampleIndexStatus
-                            + "]"
-                    )
-                );
             }
-        } else {
-            performDownsampleIndex(indexName, downsampleIndexName, ActionListener.wrap(listener::onResponse, e -> {
-                downsampleFailed = true;
-                listener.onFailure(e);
-            }));
         }
+        performDownsampleIndex(indexName, downsampleIndexName, ActionListener.wrap(listener::onResponse, e -> {
```

Comment:
Thanks for making the changes Martijn. 

I believe this needs to be in an `else` branch or above after [this line](https://github.com/elastic/elasticsearch/pull/97557/files?diff=split&w=0#diff-54833e304261a22efe3c0a185c58d4742391bd45ef654f1a0b729bcf3374a0e4R94) we need a `return;` (on line 95)